src/constants.js

```diff
@@ -315,6 +315,7 @@ export const ORIGIN_ACTIVITY_SIDEBAR: 'activity_sidebar' = 'activity_sidebar';
 export const ORIGIN_DETAILS_SIDEBAR: 'details_sidebar' = 'details_sidebar';
 export const ORIGIN_DOCGEN_SIDEBAR: 'docgen_sidebar' = 'docgen_sidebar';
 export const ORIGIN_METADATA_SIDEBAR: 'metadata_sidebar' = 'metadata_sidebar';
+export const ORIGIN_METADATA_SIDEBAR_REDESIGN: 'metadata_sidebar' = 'metadata_sidebar';
```

Comment:
this constant is the same value as the one above, is that okay? will there be conflicts anyway such as if these are used in a switch statement?

if this is intended, should we reuse the existing constant so it won't need to be changed back later?

Reply:
I've looked and they are used only for performance marks, I couldn't find where we use those though.
I decided to do the same as with `ref`s and remove this one, and reuse the `ORIGIN_METADATA_SIDEBAR`